SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -0,0 +1,108 @@
+%global debug_package %{nil}
+Summary:        NVIDIA container runtime hook
+Name:           nvidia-container-toolkit
+Version:        1.4.2
+Release:        2%{?dist}
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/NVIDIA/nvidia-container-runtime
+#Source0:       https://github.com/NVIDIA/%%{name}/archive/v%%{version}.tar.gz
+Source0:        nvidia-container-toolkit-1.4.2.tar.gz
+Source1:        vendor.tar.gz
```

Comment:
Please follow the example of `SPECS/coredns/coredns-1.6.7.spec` on how to resolve issues like these. In short: we do use tarballs the way you did, be please also provide detailed instructions how you got it and make sure the name makes it clear what the source is for.